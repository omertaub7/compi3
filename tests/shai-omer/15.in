// simple continue
void main () {
	int x = 3;
	while (x > 0) {
		x = x - 1;
		if (x == 2) {
			continue;
		}
		printi(x);
	}
	else {
		print("GOOD");
	}
}