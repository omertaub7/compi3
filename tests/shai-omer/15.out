1
0
GOOD
